### DISCUSSION TITLE
Hold shift to add a beatmap to multiple collections

### DISCUSSION THREAD

**Discussion by seemorelight** [ppy/osu#30803] [Ideas]
![Screenshot_20241120_211531](https://github.com/user-attachments/assets/c1ca1c12-bfb8-4f99-ad64-0da1c9293b02)
In the UI in the screenshot above you currently have to right click the beatmap, hover over the collections tab, and then click the collection. If you want to add a beatmap to multiple collections using that UI you would have to repeat the process over for each collection you want to add it to.

Instead, you should be able to right click the beatmap, hover over the collections tab, and then be able to click on all of the collections you would like to add the beatmap to whilst holding shift.

This would further enhance the user friendliness of the UI in lazer.

**Comment by Joehuu** [comment]
You can already right click the collections, and the menu won't close.

**Reply by seemorelight** [reply]
Oh... well then add shift too that would be double coolness

**Reply by peppy** [reply]
I'm fine with adding shift since this is the second time it's come up. Also shift is using range selection so wouldn't ctrl make more sense?

**Reply by seemorelight** [reply]
I'm not sure what you mean by using range selection, but shift feels more natural for me. I can't think of any examples of this multiple selection interface in other games, but I think they usually use shift to select multiple things at once. Ctrl could work too though.

**Reply by peppy** [reply]
Shift selects a range, ctrl selects multiple items.

Try it in any application.
